Tests/scripts/collect_tests_and_content_packs.py

```diff
@@ -1213,6 +1213,7 @@ def get_from_version_and_to_version_bounderies(all_modified_files_paths: set, id
     """
     max_to_version = LooseVersion('0.0.0')
     min_from_version = LooseVersion('99.99.99')
+    max_from_version = LooseVersion('0.0.0')
```

Comment:
Do you really need the max from version in this calculation?
Why not simply do in line 1228
```
max_to_version.vstring == '0.0.0' or max_to_version <= min_from_version
```

Reply:
Also update the method's doc string about what we're doing in that case